reputation.php

```diff
@@ -827,12 +827,19 @@
 		$neu_rep_info = $lang->neu_rep_disabled;
 	}
 
+	$perpage = (int)$mybb->settings['repsperpage'];
+	if($perpage < 1)
+	{
+		$perpage = 15;
+	}
+
 	// Check if we're browsing a specific page of results
 	if($mybb->get_input('page', MyBB::INPUT_INT) > 0)
 	{
 		$page = $mybb->get_input('page', MyBB::INPUT_INT);
-		$start = ($page-1) * $mybb->settings['repsperpage'];
 		$pages = $reputation_count / $mybb->settings['repsperpage'];
```

Comment:
Any reason you left this line? It will sitll produce FPD. Except that the PR looks ready to merge.